ai_guide/api_attractions/views.py

```diff
@@ -64,3 +66,17 @@ def create_attraction_obj(
                 attraction=attraction
             )
         return attraction
+
+
+class GetAudio(APIView):
+
+    def get(self, request, id):
+        attraction = get_object_or_404(Attraction, id=id)
+        polly = AwsPollyClient()
```

Comment:
client init shoul dlive on the app level (nexxt to open_ai_client)

Reply:
...unless it's a singleton object which is created only once and then returned every next time even by calling `AwsPollyClient()`